checker/metrics/conversion/trigger_metrics.go

```diff
@@ -138,20 +140,20 @@ func (m TriggerMetrics) Populate(lastMetrics map[string]moira.MetricState, decla
 //	{
 //	"t3": {metrics},
 //	}
-func (m TriggerMetrics) FilterAloneMetrics(declaredAloneMetrics map[string]bool) (TriggerMetrics, AloneMetrics, error) {
+func (triggerMetrics TriggerMetrics) FilterAloneMetrics(declaredAloneMetrics map[string]bool) (TriggerMetrics, AloneMetrics, error) {
 	if len(declaredAloneMetrics) == 0 {
-		return m, NewAloneMetricsWithCapacity(0), nil
+		return triggerMetrics, NewAloneMetricsWithCapacity(0), nil
 	}
 
-	result := NewTriggerMetricsWithCapacity(len(m))
-	aloneMetrics := NewAloneMetricsWithCapacity(len(m)) // Just use len of m for optimization
+	result := NewTriggerMetricsWithCapacity(len(triggerMetrics))
+	aloneMetrics := NewAloneMetricsWithCapacity(len(triggerMetrics)) // Just use len of m for optimization
```

Comment:
// Just use len of m for optimization
тут можно вместо m написать triggerMetrics, чтоб смысл не терялся

Reply:
Мне кажется, это комментарий можно вообще убрать и добавить переменную:
```suggestion
        aloneMetricsUpperBound := len(triggerMetrics)
	aloneMetrics := NewAloneMetricsWithCapacity(aloneMetricsUpperBound)
```

Reply:
Сделал немного по другому, но тоже с переменной